### ISSUE TITLE
Missing Accessibility

### ISSUE THREAD

**Issue by goelsnehaa** [Amandeepk13/nuxtAssignmentDynamoInt#12]
@Amandeepk13 

https://github.com/Amandeepk13/nuxtAssignmentDynamoInt/blob/5ae77e346132ac1f69002124fb4b8c417143220d/app/components/ApplicationTableList.vue#L105

Please make this image element accessible by providing necessary accessibility attributes. Also, review the rest of the codebase for any other image elements that may be missing accessibility support and update them accordingly.
